### ISSUE TITLE
[Error occurs in VSCode extension authoring] Navigator is not defined

### ISSUE THREAD

**Issue by nonoroazoro** [octokit/octokit.js#1526]
Version: rest.js@16.28.7~16.35.0

I've been using rest.js to access GitHub in my VSCode extension `Syncing`, but it seems something is broken since version 16.28.7. Here's the error message from VSCode dev console:

<img width="1332" alt="图片" src="https://user-images.githubusercontent.com/383132/70603612-e50c8680-1c31-11ea-84b0-9d315ff227a9.png">

Could you please help? Thanks.

**Comment by nonoroazoro** [comment]
Fixed by adding the following alias in Webpack configuration as always.

```json
alias: {
    "universal-user-agent$": "universal-user-agent/dist-node/index.js
}
```

**Comment by tanvibhakta** [comment]
For anyone coming across this in the future, this might be helpful: https://github.com/gr2m/universal-user-agent/issues/23

**Comment by gr2m** [comment]
Thanks @tanvibhakta. Yes, the problem should be resolved via gr2m/universal-user-agent#23. I'm glad you found a workaround for the time being
